rulesdoc.md

```diff
@@ -306,9 +305,18 @@ Very few creatures are able to innately cast magic. Almost everyone must prepare
 ##### Variant Rule - Spells from Codexes
 In some games spells may be limited, or codecies a rare occurance. With this variant when copying a spell from a codex, instead of destroying the codex you make an Arcana check against 2 bad dice. On success codex is not destroyed. Regardless a spell can only be copied once.
 
-### Spell Slots
+### Mana
```

Comment:
This is very similar to the SW5e system except yours is a bit more complicated.  Theirs is Spell Level + 1 is the mana necessary to cast a spell whereas you have different + X for tiers of spell levels.  I'm not sure how I feel about that, it would need playtesting.  Because they have classes, they just set the amount of mana you get per level for that class whereas you don't have classes so you have this spell power mechanic.

Reply:
I have it this way to somewhat balance high level spells. Mana gain is set in a ramp, with investing in mana early paying off over the course of your character. Because of this I need to increase the costs of higher level spells so you can't just poop out 6+ 8th-9th level spells. Doing it the SW5e way probably has the same effect, but early game Farhome mana can be a little tight.

Reply:
Spell level+1 penalizes low level spells, especially first. I like the suggested progression, but I think it should actually be something more severe, like:
| Spell Level | Mana |
|:----:|:-------------|
| 1st-2nd      | Spell level      |
| 3rd-4th      | Spell level +1 |
| 5th             | Spell level +2 | 
| 6th - 10th  | 1.5X Spell level rounded up |

Reply:
I can change things up there, although I'd probably avoid using 1.5X as that gives some weird gaps due to rounding errors.

how about something like this: 1st-3rd - Spell Level, 4th-5th Spell Level +1, 6-8th Spell Level +2, 9th-10th Spell Level +3?